tests/test_core.py

```diff
@@ -869,6 +900,31 @@ def test_delimited_list_as_string_v2(web_request, parser):
     assert data["dates"] == "2018-11-01,2018-11-02"
 
 
+@pytest.mark.skipif(
+    MARSHMALLOW_VERSION_INFO[0] < 3, reason="fields.Tuple added in marshmallow3"
+)
+def test_delimited_tuple_as_string(web_request, parser):
```

Comment:
I don't understand the purpose of the "as_string" tests.

What's in there than is not tested in the default delimiter test above?

Reply:
I think this was something of a side-effect of #464 (also explains the v2 in the other test name).
I'm looking right now to see if there's really a difference. I don't think there is.

Reply:
I compared things and this does sort of check that deserialization with `dump()` produces good results in a nontrivial case. I updated the docstring, rather than removing the tests, but I'd be comfortable removing them too. Let me know which way you prefer.

Reply:
I'd have been tempted to just merge the basic and the datetime test into one. TupleField allows to use both an IntField and a DateTimeField at once.

Having two tests is just as fine.